.rubocop.yml

```diff
@@ -1,12 +1,18 @@
 require:
   - rubocop-rails
+  - rubocop-rspec
+  - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.6
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  ExtraDetails: true
+  UseCache: true
+  CacheRootDirectory: tmp
   NewCops: disable
   Exclude:
-    - 'spec/**/*'
-    - 'db/**/*'
+    - db/schema.rb
```

Comment:
Schema is generated so not linting it makes sense. 👍 

Meanwhile we should format spec (test) and other `db` files such as migrations. 👍

Reply:
Exactly. RuboCop does more than just formatting there: rubocop-rails brings cops that lint migrations to be revertible, and more. So, its benefit is better than just format.